set.mm

```diff
@@ -261589,21 +261593,14 @@ According to Wikipedia ("Linear algebra", 03-Mar-2019,
   of linear algebra: a branch of mathematics that is concerned with
   mathematical structures closed under the operations of addition and scalar
   multiplication and that includes the theory of systems of linear equations,
-  matrices, determinants, vector spaces, and linear transformations."  However,
-  dealing with modules (over rings) instead of vector spaces (over fields)
-  allows for a more general approach. Therefore, "vectors" are regarded as
-  members (elements of the base set) of a (free) module over a ring (see
-  ~ df-frlm ) in the following.  By this, the number of entries in a vector is
-  determined by the size of the index set of the direct sum building the free
-  module the vector is belonging to. Since every vector space is isomorphic to
-  a free module (see ~ lvecisfrlm ), the theorems stated for free modules are
-  also valid for vector spaces.
-
```

Comment:
Why did you remove
```
Therefore, "vectors" are regarded as members (elements of the base set) of a (free) module over a ring
(see ~ df-frlm ) in the following.  By this, the number of entries in a vector is determined by the size
of the index set of the direct sum building the free module the vector is belonging to. 
Since every vector space is isomorphic to a free module (see ~ lvecisfrlm ), the theorems stated for
free modules are also valid for vector spaces.
```
I think this are valuable information to understand the following theorems for somebody who knows about vector spaces, but is not familiar with (left) modules.

Reply:
I added a paragraph for these readers. What I removed gives a misleading intuition. First, it focuses on free modules, and then, it gives the idea that there is always a notion of "dimension" for modules, which is not the case.